OrbitBase/include/OrbitBase/MainThreadExecutor.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_BASE_MAIN_THREAD_EXECUTOR_H_
+#define ORIBIT_BASE_MAIN_THREAD_EXECUTOR_H_
+
+#include <memory>
+#include <thread>
+
+#include "OrbitBase/Action.h"
+
+// This class implements menchanizm for landing
+// actions to a main thread. As a general rule
```

Comment:
Why "a" main thread? Can there be multiple main threads?

Reply:
there should be only one main thread. replaced 'a' with 'the'